Allow overriding ios_badges +1 in sending push

briankhdo opened the pull request:
Using merge! to override ios_badges value in default_notification_options and the new badge will be sent to devices
`other_options = {
    ios_badges: badge
}`

briankhdo committed:
Allow overriding badge count when mergin notification_options to default_notification_options

briankhdo committed:
Update gem version

briankhdo committed:
Add gem file to git

briankhdo committed:
Update version

briankhdo committed:
Ignore gem file

pedroandrade commented:
Hi @briankhdo, sorry for delayed response, but, could you remove the .gem file? and make specs pass?

briankhdo commented:
Great! Will update and re-submit soon

On Thursday, March 3, 2016, Pedro Andrade <notifications@github.com> wrote:

> Hi @briankhdo <https://github.com/briankhdo>, sorry for delayed response,
> but, could you remove the .gem file? and make specs pass?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pedroandrade/pushwoosh/pull/15#issuecomment-191719197>
> .
>


-- 
Best regards,

*Brian Doan*
*M. +84 907 244 781*

briankhdo committed:
Remove gem file

briankhdo committed:
Add spec for adding ios_badges and android_badges

notify_all_with_ios_badges_and_android_badges
notify_devices_with_ios_badges_and_android_badges
